### DISCUSSION TITLE
Support revisions appended using a dash

### DISCUSSION THREAD

**Discussion by MarcelCoding** [renovatebot/renovate#20561] [Request Help] [answered]
### What would you like Renovate to be able to do?

If a fork is maintaining versions with an original project, a revision is often prepended.
E.g: Gitea & Forgejo

Gitea:
- 1.18.3

Forgejo
- 1.18.3-0
- 1.18.3-1
- 1.18.3-2

Unfortunately, renovate can not detect this kind of updates in Forgejo.

### If you have any ideas on how this should be implemented, please tell us here.

If the other part is interpreted as major minor patch, this could be build or revision. I don't know how well this would play with suffices of flavors in docker images, maybe a check for numbers and letters would be useful.

### Is this a feature you are interested in implementing yourself?

No

**Comment by rarkins** [comment]
Try "loose" versioning

**Comment by MarcelCoding** [comment]
First of all thanks for the fast response, it theoretically it works, but now renovate tries to change the flavor: 
![image](https://user-images.githubusercontent.com/34819524/220456131-d0cb2e3c-eaef-42e1-a8cb-5f5223ef7168.png)

https://git.m4rc3l.de/marcel/stack/src/commit/175de446d9795d01621c9c01f19cc494efbfa162/renovate.json#L22-L30
https://git.m4rc3l.de/marcel/stack/pulls/143

**Comment by rarkins** [comment, marked as answer]
Combine it with allowedVersions, using a regex pattern to enforce the type of versions you need.

Also in future, please create a Discussion first instead of direct to issues
